extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/serde/Function.scala

```diff
@@ -22,7 +22,8 @@ package org.apache.kyuubi.plugin.spark.authz.serde
  *
  * Represents a function identity
  *
- * @param database
  * @param functionName
+ * @param database
+ * @param catalog
  */
-case class Function(database: Option[String], functionName: String)
+case class Function(functionName: String, database: Option[String], catalog: Option[String])
```

Comment:
According to other identity types like `Table` and `Database`, it's preferred adding `catalog` in the first place to follow the same pattern in `catalog - db/namespace - object` style.